spyder/plugins/updatemanager/plugin.py

```diff
@@ -106,11 +111,19 @@ def on_mainwindow_visible(self):
     def _populate_help_menu(self):
         """Add update action and menu to the Help menu."""
         mainmenu = self.get_plugin(Plugins.MainMenu)
+        application_plugin = self.get_plugin(Plugins.Application)
+        help_spyder_action = None
+
+        if application_plugin:
+            from spyder.plugins.application.api import ApplicationActions
+            help_spyder_action = ApplicationActions.HelpSpyderAction
```

Comment:
```suggestion
        from spyder.plugins.application.api import ApplicationActions
        help_spyder_action = ApplicationActions.HelpSpyderAction
```

The Application plugin can't be disabled, so this check is unnecessary.

Reply:
Gotcha; I'd wondered about that but I'd seen this check used for it elsewhere so I figured better be safe than sorry.